lib/src/widgets/search.dart

```diff
@@ -32,17 +39,13 @@ class ArticleSearch extends SearchDelegate<Article> {
 
   @override
   Widget buildResults(BuildContext context) {
-    var results = articles
-        .where((a) => a.title.toLowerCase().contains(query.toLowerCase()));
+    var results = articles.where((a) => a.title.toLowerCase().contains(query.toLowerCase()));
```

Comment:
Another file to be `dartfmt`'d (with 80 columns). You can run `dartfmt -w .` from the root of the project to format every Dart file at once.

Reply:
`dartfmt` did nothing here again. Addressed manually.